axiom_core.c

```diff
@@ -26,6 +26,13 @@
 #include <linux/crc16.h>
 #include <linux/slab.h>
 
+/* polling interval in ms */
+/* Based on AX54A-3D Datasheet, all touches are reported
+ * at a frame rate of up to 250Hz (4ms). However 100ms
+ * seems acceptable polling rate for human interaction.
+ * */
+#define POLL_INTERVAL_DEFAULT		100
```

Comment:
Would you mind updating the name of the define to include the time base.
e.g. `#define POLL_INTERVAL_DEFAULT_MS 100`

Could you also remove reference to the AX54A datasheet? The numbers you mention are not specific to AX54A, they apply to the whole aXiom family of devices.

Generally speaking, we aim for 100Hz - 120Hz report rate as we find that has the best perceived "feeling". However, customers and their products have have different requirements. Likewise, there are reasons why customers want to poll for touch reports, fewer interrupts.

My suggestion for the surrounding comment:
`/* aXiom devices are typically configured to report
  * touches at a rate of 100Hz (10ms). For systems
  * that require polling for reports, 100ms seems like
  * an acceptable polling rate.
  * When reports are polled, it will be expected to 
  * occasionally observe the overflow bit being set
  * in the reports. This indicates that reports are not
  * being read fast enough.
  */`

Reply:
Fixed !
Thanks,
Kamel